File: server/Employee/src/main/java/org/example/model/Employee.java
package org.example.model;

public class Employee {
    private int id;
    private String name;
    private double salary;

    // Constructeur, getters, setters...


    public Employee() {
    }

    public Employee(int id, String name, double salary) {
        this.id = id;
        this.name = name;
        this.salary = salary;
    }

    public Employee(String name, double salary) {
        this.name = name;
        this.salary = salary;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public double getSalary() {
        return salary;
    }

    public void setSalary(double salary) {
        this.salary = salary;
    }
}

File: frontend/models/Employee.js
// models/Employee.js

class Employee {
    constructor(name, salary) {
        this.name = name;
        this.salary = salary;
    }

    toXML() {
        return `<Employee>
                    <Name>${this.name}</Name>
                    <Salary>${this.salary}</Salary>
                </Employee>`;
    }
}

module.exports = Employee;
